typedb/api/answer/concept_map.py

```diff
@@ -19,7 +19,7 @@
 # under the License.
 #
 from abc import ABC, abstractmethod
-from typing import Mapping, Iterable, Tuple
+from typing import Iterable, Mapping, Tuple, Union
```

Comment:
`Iterable` is unused I think, double-check the rest of the PR for dangling unused imports